### PR TITLE
chore: thorchain remove liquidity bug fixes

### PR DESCRIPTION
**Pull request by kaladinlight**
## Description

Improvements:
- Update min withdraw limit from warning ack to disable state to prevent failed lp remove txs (4x buffer for pool asset, 2x buffer for rune)
- Omit asym asset id on non applicable txs (only applicable for sym position/asym withdraw)
- Localize rune outbound fee const and return from hook
- More explicit variable names
- Reduce `isDisabled` check to just include `errorCopy` instead of all the individual states `errorCopy` already checks

Bug fixes:
- Only estimate rune fees for Sym and Asym Rune positions and pool asset fees for Asym Asset positions (currently asym withdraw on sym position for utxo coins are broken)
- Pool asset dust amount should be `0` for Sym and Asym Rune positions
- Only check `isSweepNeeded` for Asym Asset positions
- Only check pool asset fee asset balance for Asym Asset positions
- Only check rune fee asset balance for Sym and Asym Rune positions
- Loading and disabled states correspond to the correct opportunity type

## Issue (if applicable)

closes https://github.com/shapeshift/web/issues/7710

## Risk

> High Risk PRs Require 2 approvals

Medium - several changes surrounding fixed estimations and displayed fees to be validated

> What protocols, transaction types, wallets or contract interactions might be affected by this PR?

## Testing

- Ensure the confirm button is disabled if below the min withdraw limit, and that withdraws are successful when able to be confirmed
- Ensure all withdraw states load correctly and show the correct fees
  - Asym Asset: transaction is on asset chain and fees are for asset chain with slippage displayed
  - Asym Rune and Sym: transaction is on rune and fees are for thorchain with slippage only displayed on asym withdraws

### Engineering

:point_up:

### Operations

- [ ] :checkered_flag: My feature is behind a flag and doesn't require operations testing (yet)

:point_up:

### Sym Position (EVM)

Minimum Withdraw Disable State
image

Gas fee should be in rune, not pool asset for asym asset withdraw
image
image

### Sym Position (UTXO)

Gas fee should be in rune, not pool asset and rune for sym withdraw
image

Gas fee should be in rune, not pool assest and rune for asym rune withdraw
image
image

You can now successfully perform an asym asset withdraw
image

### Asym Rune Position

Gas fee should be in rune, not showing both rune and pool asset fee
image
image

### CODE REVIEW

File: src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx [hunk 1]
```diff
@@ -892,29 +890,63 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
...
+    const poolAssetWithrawAmountFiatUserCurrency = bnOrZero(
       actualAssetWithdrawAmountFiatUserCurrency,
-    ).plus(bnOrZero(actualRuneWithdrawAmountFiatUserCurrency))
+    )
+    const runeWithrawAmountFiatUserCurrency = bnOrZero(actualRuneWithdrawAmountFiatUserCurrency)
+
+    if (
+      poolAssetWithrawAmountFiatUserCurrency.gt(0) &&
+      minimumPoolAssetWithdrawAmountFiatUserCurrency.gte(poolAssetWithrawAmountFiatUserCurrency) &&
+      minimumPoolAssetWithdrawAmountFiatUserCurrency.gt(minimumRuneWithdrawAmountFiatUserCurrency)
```
Comment (NeOMakinG) [inline src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx:919]: suggestion: I took a little bit of time to understand this condition and why we need the latest part, maybe we could put this in a `isPoolAssetWithdrawAmountTooSmall` or something similar?

**Review by NeOMakinG — COMMENTED** [review]
Functional review, pending e2e tests

**Review by NeOMakinG — APPROVED** [review]
## Sym position (evm)
image
Gas fees
image

## Asym rune position
image

## SYM UTXO
image
image
image
image

I've been able to withdraw sym choosing utxo

Looks sane so far

File: src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx [hunk 2]
```diff
@@ -892,29 +890,63 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
...
+    opportunityType,
     runeAsset,
     runeBalanceCryptoBaseUnit,
     runeDustAmountCryptoBaseUnit,
     runeTxFeeCryptoPrecision,
   ])
 
-  const isBelowMinimumWithdrawAmount = useMemo(() => {
-    const totalWithdrawAmountFiatUserCurrency = bnOrZero(
+  // Protocol fee buffers explained here: https://gitlab.com/thorchain/thornode/-/blob/develop/x/thorchain/querier_quotes.go#L461
```
[the remark this answers is not in the document]
Comment (kaladinlight) [reply src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx:900]: yeah lots of black magic...

File: src/lib/utils/thorchain/constants.ts [hunk 3]
```diff
@@ -4,6 +4,7 @@ export const THOR_PRECISION = 8
 export const BASE_BPS_POINTS = '10000'
 export const THORCHAIN_AFFILIATE_NAME = 'ss'
 export const THORCHAIN_POOL_MODULE_ADDRESS = 'thor1g98cy3n9mmjrpn0sxmn63lztelera37n8n67c0'
+export const THORCHAIN_OUTBOUND_FEE_CRYPTO_BASE_UNIT = '2000000'
```
[the remark this answers is not in the document]
Comment (kaladinlight) [reply src/lib/utils/thorchain/constants.ts:7]: Those are in packages so shouldn't be reaching into web/src for them

File: src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx [hunk 4]
```diff
@@ -430,15 +426,23 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
     action: 'withdrawLiquidity',
   })
 
-  const poolAssetProtocolFeeCryptoPrecision = useMemo(() => {
-    if (!poolAssetFeeAsset || !outboundFeeCryptoBaseUnit) return bn(0)
+  console.log({ isEstimatedRuneFeesDataLoading, isEstimatedPoolAssetFeesDataLoading })
```
[the remark this answers is not in the document]
Comment (kaladinlight) [reply src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx]: 8801490c03b1d85b63abc966015de98346cffde5

File: src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx (hunk 1 above)
Comment (kaladinlight) [reply src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx:919]: added some comments for clarity: 69b9605579939a0ae20d77a1bbde96a628de56e9

**Pull request merged**